go/chat/ephemeral_purger.go

```diff
@@ -158,21 +164,29 @@ func (b *BackgroundEphemeralPurger) Start(ctx context.Context, uid gregor1.UID)
 			storage.NewOutbox(b.G(), b.uid).EphemeralPurge(context.Background())
```

Comment:
It's a little sloppy that this loop doesn't get shutdown.

Reply:
done!